diff --git a/modest_image/modest_image.py b/modest_image/modest_image.py
index ef257b9..548d86b 100644
--- a/modest_image/modest_image.py
+++ b/modest_image/modest_image.py
@@ -52,12 +52,12 @@ class ModestImage(mi.AxesImage):
         self._A = A
 
         if self._A.dtype != np.uint8 and not np.can_cast(self._A.dtype,
-                                                         np.float):
+                                                         float):
             raise TypeError("Image data can not convert to float")
 
         if (self._A.ndim not in (2, 3) or
                 (self._A.ndim == 3 and self._A.shape[-1] not in (3, 4))):
-                raise TypeError("Invalid dimensions for image data")
+            raise TypeError("Invalid dimensions for image data")
 
         self.invalidate_cache()
 
